libexec/rtld-elf/rtld.c

```diff
@@ -238,6 +238,9 @@ static bool ld_skip_init_funcs = false;	/* XXXAR: debug environment variable to
 static struct obj_entry_q obj_list;	/* Queue of all loaded objects */
 static Obj_Entry *obj_main;	/* The main program shared object */
 static Obj_Entry obj_rtld;	/* The dynamic linker shared object */
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(RTLD_SANDBOX)
+const Obj_Entry *obj_rtld_p = &obj_rtld;
```

Comment:
I'm curious why you added an intermediate pointer vs removing static from `obj_rtld`?  Was this trying to reduce future merge conflicts?  I think it would be fine to do:

```
#if !(defined(__CHERI_PURE_CAPABILITY__) && defined(RTLD_SANDBOX))
static
#endif
Obj_Entry obj_rtld;
```

Reply:
Yes. Although now it looks like this new variable isn't used anymore. So I'll remove it. Thanks for pointing this out.